### PR TITLE
Remove runtime dependencies on core

### PR DESCRIPTION
**Pull request by lburgazzoli**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -347,13 +351,13 @@ workflows:
               only: master
       - license-check
       - ui
-      - runtime
       - core:
+      - runtime
           requires:
-            - runtime
+            - core
```
Comment (rhuss) [inline .circleci/config.yml]: don't think that runtime requires core now (it was just @chirino's PR, but I hope that we can avoid tha)

File: .circleci/config.yml [hunk 2]
```diff
@@ -347,13 +351,13 @@ workflows:
               only: master
       - license-check
       - ui
-      - runtime
       - core:
```
Comment (rhuss) [inline .circleci/config.yml]: colon ?

File: .circleci/config.yml [hunk 3]
```diff
@@ -347,13 +351,13 @@ workflows:
...
+      - runtime
```
Comment (rhuss) [inline .circleci/config.yml]: colon !

File: tools/bin/commands/build [hunk 4]
```diff
@@ -1,17 +1,17 @@
...
   "ui"
-  "runtime"
-  "core:runtime"
-  "connectors:runtime core"
-  "verifier:runtime core connectors"
-  "rest:runtime core connectors"
-  "s2i:runtime core connectors rest"
-  "tests:runtime core connectors rest s2i"
+  "core"
+  "runtime:core"
```
Comment (rhuss) [inline tools/bin/commands/build]: would remove core for now here

File: .circleci/config.yml [hunk 5]
```diff
@@ -347,13 +351,13 @@ workflows:
               only: master
       - license-check
       - ui
-      - runtime
-      - core:
+      - core
+      - runtime:
           requires:
-            - runtime
+            - core
```
Comment (rhuss) [inline .circleci/config.yml]: runtime does not require core (introduced only in hirams PR)

Comment (lburgazzoli) [reply .circleci/config.yml]: check now

**Pull request merged**